src/stores/SettingsStore.ts

```diff
@@ -25,6 +26,7 @@ const defaultValues: SettingsType = {
   affectsPerPage: 10,
   trackersPerPage: 10,
   isHidingLabels: false,
+  privacyNoticeShown: false,
 };
 const osimSettings = useStorage('OSIM::USER-SETTINGS', structuredClone(defaultValues));
 
```

Comment:
Potentially we should try to remove the extra settings ref (replace `osimSettings` with just `settings` outside the store) and just modify that directly

Reply:
that way we could remove the unused save function and the watcher along with the settings ref

Reply:
we can leave it in the store since that's basically global anyway